actions/publish-github-pages/action.yml

```diff
@@ -1,10 +1,17 @@
 name: 'Publish GitHub Pages'
 description: 'Publish the Hugo-based CloudPosse docs website to GitHub Pages.'
 runs:
-#  using: 'docker'
-#  image: 'Dockerfile'
   using: 'composite'
   steps:
-    - run: ${{ github.action_path }}/entrypoint.sh
+#    - uses: actions/setup-python@v2
+#      with:
+#        python-version: '3.x'
+    - run: |
+        export STAGING_DIR=/tmp/staging/
+        export GITHUB_PAGES_PUSH_PATH=$(pwd)/${GITHUB_PAGES_DIRECTORY}
```

Comment:
Should this instead be an `env` section?

Reply:
I rearranged a few things, but left this in place. If it still feels awkward, I can move it to the GitHub Actions `env` section.